containers/daap-python-base/tests/unit/data_platform_paths_test.py

```diff
@@ -39,79 +41,115 @@ def test_bucket_path_parent():
     assert path.parent.key == "path/to"
 
 
-def test_bucket_name_returns_environment_variable(monkeypatch):
-    monkeypatch.setenv("BUCKET_NAME", "a-bucket")
-    assert get_bucket_name() == "a-bucket"
+def test_raw_data_bucket(monkeypatch):
+    monkeypatch.setenv("RAW_DATA_BUCKET", "a-bucket")
+    assert get_raw_data_bucket() == "a-bucket"
+
+
+def test_curated_data_bucket(monkeypatch):
+    monkeypatch.setenv("CURATED_DATA_BUCKET", "a-bucket")
+    assert get_curated_data_bucket() == "a-bucket"
+
+
+def test_landing_zone_bucket(monkeypatch):
+    monkeypatch.setenv("LANDING_ZONE_BUCKET", "a-bucket")
+    assert get_landing_zone_bucket() == "a-bucket"
+
+
+def test_log_bucket(monkeypatch):
+    monkeypatch.setenv("LOG_BUCKET", "a-bucket")
+    assert get_log_bucket() == "a-bucket"
+
+
+def test_metdata_bucket(monkeypatch):
+    monkeypatch.setenv("METADATA_BUCKET", "a-bucket")
+    assert get_metadata_bucket() == "a-bucket"
 
 
-def test_data_product_metadata_file_path(monkeypatch):
+def test_raw_data_bucket_defaults_to_old_environment_variable(monkeypatch):
     monkeypatch.setenv("BUCKET_NAME", "a-bucket")
-    metadata_path = data_product_metadata_file_path("delicious-data-product")
+    assert get_raw_data_bucket() == "a-bucket"
 
-    assert (
-        metadata_path
-        == "s3://a-bucket/metadata/delicious-data-product/v1.0/metadata.json"
+
+def test_data_product_config_metadata_spec_prefix():
+    assert DataProductConfig.metadata_spec_prefix("bucket") == BucketPath(
+        bucket="bucket", key="data_product_metadata_spec"
     )
 
 
-def test_data_product_raw_data_file_path():
-    uuid_value = uuid.uuid4()
-    timestamp = datetime(2023, 9, 6)
-    path = data_product_raw_data_file_path(
-        data_product_name="data-product",
-        table_name="table",
-        bucket_name="bucket",
-        extraction_timestamp=timestamp,
-        uuid_value=uuid_value,
-    )
+def test_data_product_element_config(monkeypatch):
+    monkeypatch.setenv("RAW_DATA_BUCKET", "bucket")
+    monkeypatch.setenv("CURATED_DATA_BUCKET", "bucket")
+    monkeypatch.setenv("METADATA_BUCKET", "bucket")
+    monkeypatch.setenv("LANDING_ZONE_BUCKET", "bucket")
 
-    assert (
-        path
-        == f"s3://bucket/raw_data/data-product/table/extraction_timestamp=20230906T000000Z/{uuid_value}"
+    element = DataProductElement.load("some-table", "data-product")
+
+    raw_data_table = element.raw_data_table_unique()
+
+    assert re.match(
+        r"some-table_[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}_raw",
```

Comment:
Note: not compatible (as is) with my suggestion for shorter UUIDs (not difficult to add though)